app/lib/menu.rb

```diff
@@ -13,8 +13,8 @@ def initialize
     @items = {}
   end
 
-  def scope(name, children)
-    @items[name] = children
+  def scope(name, svg, children)
+    @items[name] = { svg: svg, children: children }
```

Comment:
How about this? If we know all the icons have to be SVGs we can simplify menu creation by adding the file extension here:

```suggestion
    @items[name] = { svg: "#{svg}.svg", children: children }
```